src/main/kotlin/com/getkeepsafe/dexcount/Deobfuscator.kt

```diff
@@ -26,10 +26,9 @@ import java.util.TreeMap
  * mapping file.
  */
 open class Deobfuscator(reader: MappingReader?) {
-    private val mapping: MutableMap<String, String>
+    private val mapping = TreeMap<String, String>()
```

Comment:
I generally prefer to leave out type annotations where I can, especially on private properties.  I also prefer to keep `init` blocks as small as possible.  In fact, I kind of want to do a larger refactoring to remove the remaining statement in there.  Didn't do it here because I'm already spending too much time on the bike-shed :)